stable/filebeat/values.yaml

```diff
@@ -145,6 +145,17 @@ podSecurityPolicy:
 ## Dont forget to enable http on config.http.enabled (exposing filebeat stats)
 monitoring:
   enabled: true
+  serviceMonitor:
+    # When set true and if Prometheus Operator is installed then use a ServiceMonitor to configure scraping
+    enabled: true
+    # Set the namespace the ServiceMonitor should be deployed
+    namespace: monitoring
+    # Set how frequently Prometheus should scrape
+    interval: 30s
+    # Set labels for the ServiceMonitor, use this to define your scrape label for Prometheus Operator
+    # labels:
+    # Set path to beats-exporter telemtery-path
+    telemetryPath: /metrics
```

Comment:
Why have this key `telemetryPath` be different than just the default `path` that is already in the serviceMonitor spec?

Reply:
I made it possible to override this just in case someone changed `-web.telemetry-path` on the exporter via the `args` value however looking at this retrospectively it feels like overkill so I will improve this when back at my dev machine. Thanks for the suggestion.

Reply:
Pushed a new commit to remove `telemetryPath` 👍